tests/Behat/Resources/suites/ui/product/managing_product_variants.yaml

```diff
@@ -0,0 +1,34 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_managing_product_variants:
+            contexts:
+                - sylius.behat.context.hook.doctrine_orm
+
+                - sylius.behat.context.transform.channel
+                - sylius.behat.context.transform.lexical
+                - sylius.behat.context.transform.locale
+                - sylius.behat.context.transform.product
+                - sylius.behat.context.transform.product_variant
+                - sylius.behat.context.transform.shared_storage
+                - sylius.behat.context.transform.taxon
+
+                - sylius.behat.context.setup.admin_security
+                - sylius.behat.context.setup.admin_user
+                - sylius.behat.context.setup.channel
+                - sylius.behat.context.setup.locale
+                - sylius.behat.context.setup.product
+                - sylius.behat.context.setup.shop_security
+                - sylius.behat.context.setup.taxonomy
+                - Sylius\Behat\Context\Setup\CatalogPromotionContext
+                - Tests\Sylius\PriceHistoryPlugin\Behat\Context\Setup\CatalogPromotionContext
+                - Tests\Sylius\PriceHistoryPlugin\Behat\Context\Setup\ProductContext
```

Comment:
I'm fairly certain some of those are not needed, same case with `ui/promotion/managing_catalog_promotions.yaml`